### DISCUSSION TITLE
Google OR Tools crashes with GRPC C++

### DISCUSSION THREAD

**Discussion by parthjindal** [google/or-tools#3659] [Routing (and legacy CP) questions]
**What version of OR-Tools and what language are you using?**
Version: v9.5
Language: C++

**Which solver are you using?**
 Routing Solver

**What operating system (Linux, Windows, ...) and version?**
Ubuntu 22

**What did you do?**
I am trying to create a c++ project that uses google or-tools for routing problem solver. This project would communicate with other projects via grpc c++. I am using grpc v1.50 and building it from source. For google or-tools I have used the binary for v9.5 available for Ubuntu 22.04. 

**What did you expect to see?**  
A solved route as response to my grpc request.

**What did you see instead?**
The thread handling the request just exits as soon as `RoutingModel routing(manager)` gets called

**Comment by lperron** [comment]
Can you check the routing code works outside of grpc?

Most likely not.
